Regular-Question-Answers/minimum-path-sum.cpp

```diff
@@ -0,0 +1,31 @@
+	// Ali Davut Eskiocak
+	//https://leetcode.com/problems/minimum-path-sum/
+	
+	class Solution {
+	public:
+	//we will create a 2 D array and we will store most efficient cost to reach each position
+	//we will use dynamic memory approach 
+	//Time complexity 0(n*m)
+	//Space complexity o(1)

```

Comment:
Mentioning about how you will proceed throughout your code is a nice introduction to explain your approach. Also you write down the time and space complexity of your code but maybe in this part or in for loops you might have talk about how you calculated the complexities.